File: se321/se321-pz-nikola_tasic_3698/src/main/java/com/example/backend/entity/Category.java
package com.example.backend.entity;

import lombok.Data;
import lombok.EqualsAndHashCode;

import javax.persistence.*;

/**
 * Blog post category
 */
@Data
@Entity
@Table(name = "category")
@EqualsAndHashCode(callSuper = false, onlyExplicitlyIncluded = true)
public class Category extends Auditable {
	@Id
	@GeneratedValue(strategy = GenerationType.IDENTITY)
	@EqualsAndHashCode.Include
	@Column(name = "category_id")
	private Integer id;
	@Column(name = "name")
	private String name;
}

File: cs322/cs322-pz-nikola_tasic_3698/blog-backend/Data/Entity/Category.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace blog_backend.Data;

[Table("category")]
public class Category : Auditable {
	[Key, Column("category_id")]
	public int Id { get; set; }
	[Column("name")]
	public string Name { get; set; }
}